pom.xml

```diff
@@ -22,7 +22,7 @@
     <jib-maven-plugin.version>3.4.2</jib-maven-plugin.version>
     <structured-logging.version>3.0.13</structured-logging.version>
     <api-sdk-manager-java-library.version>3.0.5</api-sdk-manager-java-library.version>
-    <private-api-sdk-java.version>4.0.187</private-api-sdk-java.version>
+    <private-api-sdk-java.version>unversioned</private-api-sdk-java.version>
```

Comment:
This needs the real version

Reply:
Changes are here https://github.com/companieshouse/private.api.ch.gov.uk-specifications/pull/367